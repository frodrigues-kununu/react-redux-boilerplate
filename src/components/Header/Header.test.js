import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import toJSON from 'enzyme-to-json';
import Header from './Header';

Enzyme.configure({ adapter: new Adapter() });

/*****************************
* Profile chart Selected Anomaly Visibility
*/
function setup() {
  const props = {};

  const enzymeWrapper = shallow(<Header {...props} />);

  return {
    props,
    enzymeWrapper,
  };
}

//test wrapper and props
const { enzymeWrapper, props } = setup();

describe('<Header />', () => {
  //snapshot testing
  test('should render correctly', () => {

    expect(toJSON(enzymeWrapper)).toMatchSnapshot();
  });
});
