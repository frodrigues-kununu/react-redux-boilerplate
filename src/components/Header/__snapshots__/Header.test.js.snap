// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<header
  className="page-header"
>
  <nav>
    <Link
      replace={false}
      to="/"
    >
      Home
    </Link>
    <Link
      replace={false}
      to="/about"
    >
      About
    </Link>
  </nav>
</header>
`;
